### PR TITLE
fornax-serverles install doc detail

### PR DESCRIPTION
**Pull request by davidzhu22**
fornax-serverles install doc detail

### CODE REVIEW

File: doc/fornax_setup.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+This doc is the detail steps which setting up, configuration and install component for Fornax Serverless Machine.
+
+## Machine Prepare
+If you have a brand new machine, your should install following component to start the machine. (Following is detail steps for AWS EC2 virtual machine).
+
+```script
+sudo apt-get update
+sudo apt install build-essential
+sudo snap install curl  # version 7.81.0
+sudo apt-get install vim
```
Comment (guowenjian90) [inline doc/fornax_setup.md:13]: why do we need vim and build-essential, it's not for developer

File: doc/fornax_setup.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+```
+
+```sh
+export GOPATH="/home/ubuntu/go"
+export GOROOT="/usr/local/go"
+export GOBIN="/home/ubuntu/go/bin"
+export PATH="$PATH:/usr/local/go/bin:/home/ubuntu/go/bin"
+```
+Take effetive
+```script
```
Comment (guowenjian90) [inline doc/fornax_setup.md:72]: I do not think these details are necessary, any go developer should know how to do this. it distract reader

File: doc/fornax_setup.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+sudo systemctl status docker
+```
+
+#### 2.1.2 Enable Containerd CRI Plugins
+
+Find containerd config.toml file
+```script
+dpkg -L containerd.io | grep toml
+```
+Using vi to open the file
```
Comment (guowenjian90) [inline doc/fornax_setup.md:123]: why must vi, people can use any editor they like to use

File: doc/fornax_setup.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+```
+In centaurusinfra.io folder and run follwing command
+```script
+git clone https://github.com/CentaurusInfra/fornax-serverless.git
+cd fornax-serverless
+```
+You are in your workplace folder: fornax-serverless
+
+```script
+sudo make all
```
Comment (guowenjian90) [inline doc/fornax_setup.md:90]: why sudo? sudo is to get root user permission

**Review by guowenjian90 — COMMENTED** [review]
I think this doc meant to tell people how to install and use, not teach them how to develop, so, do not include too many details developer are supposed to know.

File: doc/fornax_setup.md (hunk 1 above)
Comment (davidzhu22) [reply doc/fornax_setup.md:13]: If new virtual machine created by AWS (brand new VM), we need install this build-essential and vim. sometime default vim is old version with VM

File: doc/fornax_setup.md (hunk 4 above)
Comment (davidzhu22) [reply doc/fornax_setup.md:90]: I can take sudo off

File: doc/fornax_setup.md (hunk 2 above)
Comment (davidzhu22) [reply doc/fornax_setup.md:72]: Yes, your are right, But some entry level user, lower intermedia level user, and site admin need more detail instruction. I thought that we keep your get_start.md as Professional user use. the user can reference fornax_setup.md as detail if they need.

File: doc/fornax_setup.md (hunk 3 above)
Comment (davidzhu22) [reply doc/fornax_setup.md:123]: You are right, I can change to "You can use vi or  other tool to open the file"

**Pull request merged**